src/components/field.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+
+import styles from "./field.module.scss"
+import classNames from "classnames"
+
+
+interface Props {
+    className?: string
+    labelText?: string
+    labelFor?: string
+    contextualHelp?: string
+    error: string | undefined
+    children: React.ReactNode
```

Comment:
don't need this -  TypeScript includes the children prop if you're typing it `React.FC`, which you are.